chain/chain/src/chain.rs

```diff
@@ -76,6 +76,12 @@ pub const MAX_ORPHAN_SIZE: usize = 1024;
 /// Maximum age of orhpan to store in the chain.
 const MAX_ORPHAN_AGE_SECS: u64 = 300;
 
+// Number of orphan ancestors should be checked to request chunks
+const NUM_ORPHAN_ANCESTORS_CHECK: u64 = 5;
+
+// Maximum number of orphans that we can request missing chunks
+const MAX_ORPHAN_MISSING_CHUNKS: usize = 100;
```

Comment:
how are these number set?

Reply:
The second number is somewhat random, just to set a max number and that should almost never be hit, unless the chain has many forks at the same time. 

The first number was set because both ChunkCache(stores processed PartialEncodedChunks) and stored_partial_encoded_chunks(stores unprocessed PartialEncodedChunks) only stores chunks within 5 blocks of the current head, so requesting something larger than that doesn't make sense.

Looking at the chunks delay metric provided by Nikolay, it may make sense to change both of these numbers to 10, because that's usually how much a node is behind. But I think we can try 5 now and if that doesn't solve the issue, we can make the number higher.